<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>585</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>585</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>绝望2005第三方登陆系统</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>src/logo_school.png</normaloff>src/logo_school.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>585</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>585</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>585</height>
     </size>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="mainPage">
     <widget class="QFrame" name="mainFrame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1000</width>
        <height>585</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>170</y>
         <width>361</width>
         <height>251</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="welcomePageLabel">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>170</y>
         <width>371</width>
         <height>241</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>绝望2005第三方登陆系统欢迎你~</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="userWelcomeLabel">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>200</y>
         <width>291</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="schoolLogolabel">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>801</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>src/long_logo_school.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="loadingPage">
     <widget class="QFrame" name="loadingFrame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1000</width>
        <height>585</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="loadingLabel">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>130</y>
         <width>591</width>
         <height>180</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Kristen ITC</family>
         <pointsize>36</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Loading</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>240</y>
         <width>520</width>
         <height>30</height>
        </rect>
       </property>
       <property name="value">
        <number>33</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="textDirection">
        <enum>QProgressBar::TopToBottom</enum>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="cjPage">
     <widget class="QFrame" name="cjFrame">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1000</width>
        <height>585</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>960</width>
        <height>540</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1000</width>
        <height>585</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QGroupBox" name="cjGroupBox1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>980</width>
         <height>110</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>980</width>
         <height>110</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="userImformationLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>951</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;学院 专业  姓名学号&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="buddy">
         <cstring>cjTabelWidget</cstring>
        </property>
       </widget>
       <widget class="QLabel" name="xnOpptionLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>学年</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buddy">
         <cstring>xnOpptionComboBox</cstring>
        </property>
       </widget>
       <widget class="QComboBox" name="xnOpptionComboBox">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>73</y>
          <width>131</width>
          <height>25</height>
         </rect>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>2005-2006</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2006-2007</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2007-2008</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2008-2009</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2009-2010</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2010-2011</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2011-2012</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2012-2013</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2013-2014</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="xqOpptionLabel">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>70</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>学期</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buddy">
         <cstring>xqOpptionComboBox</cstring>
        </property>
       </widget>
       <widget class="QComboBox" name="xqOpptionComboBox">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>73</y>
          <width>131</width>
          <height>25</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>全部学期</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="lnCjPushButton">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>70</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>历年成绩</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="cjGroupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>980</width>
         <height>360</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>980</width>
         <height>360</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>成绩</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <widget class="QTableView" name="cjTabelWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>960</width>
          <height>331</height>
         </rect>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>true</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::DashDotDotLine</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="userGPALabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>490</y>
         <width>980</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>GPA                                                                    智育</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="kbPage">
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1000</width>
        <height>585</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QComboBox" name="kbXnComboBox">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>10</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="kbXqComboBox">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>10</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="kbXnLabel">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>10</y>
         <width>80</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>学年</string>
       </property>
       <property name="buddy">
        <cstring>kbXnComboBox</cstring>
       </property>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>980</width>
         <height>500</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>一周课表</string>
        </attribute>
        <widget class="QTableView" name="kbTableView">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>975</width>
           <height>480</height>
          </rect>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideMiddle</enum>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>实践课信息</string>
        </attribute>
        <widget class="QTableView" name="sjkTableView">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>50</y>
           <width>980</width>
           <height>230</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>实习课信息</string>
        </attribute>
        <widget class="QTableView" name="sxkTableView">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>50</y>
           <width>980</width>
           <height>230</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>未安排上课时间的课程</string>
        </attribute>
        <widget class="QTableView" name="wapTableView">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>50</y>
           <width>980</width>
           <height>230</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QLabel" name="kbXqLabel">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>10</y>
         <width>80</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>学期</string>
       </property>
       <property name="buddy">
        <cstring>kbXqComboBox</cstring>
       </property>
      </widget>
      <widget class="QLabel" name="kbImformationLabel">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>10</y>
         <width>200</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>谁谁谁的课表</string>
       </property>
       <property name="buddy">
        <cstring>kbTableView</cstring>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="BrowerserMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="xyMenu">
    <property name="title">
     <string>信息查询</string>
    </property>
    <addaction name="cjAction"/>
    <addaction name="kbAction"/>
   </widget>
   <widget class="QMenu" name="xkMenu">
    <property name="title">
     <string>选课</string>
    </property>
    <addaction name="xsxkAction"/>
   </widget>
   <widget class="QMenu" name="syMenu">
    <property name="title">
     <string>首页</string>
    </property>
    <addaction name="zyAction"/>
    <addaction name="separator"/>
    <addaction name="logoutAction"/>
   </widget>
   <addaction name="syMenu"/>
   <addaction name="xyMenu"/>
   <addaction name="xkMenu"/>
  </widget>
  <action name="cjAction">
   <property name="text">
    <string>成绩查询</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="kbAction">
   <property name="text">
    <string>课表查询</string>
   </property>
  </action>
  <action name="xsxkAction">
   <property name="text">
    <string>学生选课</string>
   </property>
  </action>
  <action name="zyAction">
   <property name="text">
    <string>首页</string>
   </property>
  </action>
  <action name="logoutAction">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>calendarWidget</tabstop>
  <tabstop>xnOpptionComboBox</tabstop>
  <tabstop>xqOpptionComboBox</tabstop>
  <tabstop>lnCjPushButton</tabstop>
  <tabstop>cjTabelWidget</tabstop>
  <tabstop>kbXnComboBox</tabstop>
  <tabstop>kbXqComboBox</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>kbTableView</tabstop>
  <tabstop>sjkTableView</tabstop>
  <tabstop>sxkTableView</tabstop>
  <tabstop>wapTableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
